src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -61,25 +63,75 @@ def test_help_arguments_output_and_exit(
     @pytest.mark.parametrize(
         "arg, expected_attributes",
         [
+            (["--concurrency", "3"], {"concurrency_range": "3"}),
+            (["--endpoint", "v1/completions"], {"endpoint": "v1/completions"}),
+            (
+                ["--endpoint", "v1/chat/completions"],
+                {"endpoint": "v1/chat/completions"},
+            ),
+            (["--expected-output-tokens", "5"], {"expected_output_tokens": 5}),
+            (["--input-tokens-mean", "6"], {"input_tokens_mean": 6}),
+            (["--input-tokens-stddev", "7"], {"input_tokens_stddev": 7}),
+            (
+                ["--input-type", "file"],
+                {"input_type": utils.get_enum_entry("file", InputType)},
+            ),
+            (
+                ["--input-type", "synthetic"],
+                {"input_type": utils.get_enum_entry("synthetic", InputType)},
+            ),
+            (
+                ["--input-type", "url"],
+                {"input_type": utils.get_enum_entry("url", InputType)},
+            ),
+            (["--measurement-interval", "100"], {"measurement_interval": 100}),
+            (["-p", "100"], {"measurement_interval": 100}),
+            (["--num-of-output-prompts", "101"], {"num_of_output_prompts": 101}),
+            (
+                ["--output-format", "openai_chat_completions"],
+                {
+                    "output_format": utils.get_enum_entry(
+                        "openai_chat_completions", OutputFormat
+                    )
+                },
+            ),
+            (
+                ["--output-format", "openai_completions"],
+                {
+                    "output_format": utils.get_enum_entry(
+                        "openai_completions", OutputFormat
+                    )
+                },
+            ),
+            (
+                ["--output-format", "trtllm"],
+                {"output_format": utils.get_enum_entry("trtllm", OutputFormat)},
+            ),
+            (
+                ["--output-format", "vllm"],
+                {"output_format": utils.get_enum_entry("vllm", OutputFormat)},
+            ),
             (
                 ["--profile-export-file", "text.txt"],
                 {"profile_export_file": Path("text.txt")},
             ),
+            (["--random-seed", "8"], {"random_seed": 8}),
+            (["--request-rate", "9.0"], {"request_rate_range": "9.0"}),
             (["--service-kind", "triton"], {"service_kind": "triton"}),
             (["--service-kind", "openai"], {"service_kind": "openai"}),
+            (["--stability-percentage", "99.5"], {"stability_percentage": 99.5}),
+            (["-s", "99.5"], {"stability_percentage": 99.5}),
+            (["--streaming"], {"streaming": True}),
             (["--version"], {"version": True}),
-            (["-u", "test_url"], {"u": "test_url"}),
+            (["-v"], {"version": True}),
```

Comment:
This will break when you undo the change in the parser.py file.

Reply:
Thanks for the heads up. Test is passing.

![image](https://github.com/triton-inference-server/client/assets/58150256/36a16760-3f0d-40a0-872c-228ed04abd22)